participants/_template.json

```diff
@@ -1,16 +1,36 @@
 {
-	"name": "My Name",
-	"company": "my company",
+	// The schema which is used for validation (see https://json-schema.org/)
+	"$schema": "./_schema.json",
+	// your name / nickname
+	"name": "John Doe",
+	// optional company name
+	"company": "ACME",
+	// either both days or at least one day has to be set to true
 	"when": {
+		// 2023-06-30
 		"friday": true,
+		// 2023-07-01
 		"saturday": false
 	},
+	// your current interests (JS and in general)
 	"tags": ["ReScript", "Svelte", "Zustand", "GraphQL"],
+	// if you only eat vegan food
 	"vegan": false,
-	"vegetarian": false,
-	"allergies": "",
-	"what_is_my_connection_to_javascript": "...",
-	"what_can_i_contribute": "???",
-	"tshirt": "/^(W|M)-(S|M|L|XL)$/",
-	"twitter": ""
+	// if you only eat vegan or vegetarian food
+	"vegetarian": true,
+	// what you cannot eat or drink
+	"allergies": ["🥜", "🍏"],
```

Comment:
Let's add text or these examples or it will be hard for us to interprete by emojis later.